src/components/ContentModal.vue

```diff
@@ -11,6 +11,7 @@
         class="img-responsive"
         v-if="isImage"
       />
+      <div id="markdown" v-if="isMarkdown" v-html="markdownContent"></div>
```

Comment:
```suggestion
      <div id="markdown" v-else-if="isMarkdown" v-html="markdownContent"></div>
```
Would prefer v-else-if instead of multiple if's

Reply:
I will change this to v-else-if